### PR TITLE
Install celery, configure for local usage only.

### PR DESCRIPTION
**Pull request by iloveagent57**
https://openedx.atlassian.net/browse/EDUCATOR-4113

* Installs celery into the project.
* Sets things up for local task invocation only.
* Adds a simple `debug_task`.
* Adds a unit test of the debug_task to make sure that we can appropriately unit test tasks.

### CODE REVIEW

File: registrar/settings/base.py [hunk 1]
```diff
@@ -80,6 +80,31 @@
     }
 }
 
+############################# END CELERY #################################3
```
Comment (kdmccormick) [inline registrar/settings/base.py]: `### BEGIN CELERY ###`

**Review by kdmccormick — APPROVED** [review]
LGTM, pending one nit

**Comment by kdmccormick** [comment]
The ticket has an acceptance criteria "Update settings value to specify Celery broker and queue name". I assume this isn't relevant? Or maybe I missed the part of the code that does this?

**Comment by iloveagent57** [comment]
Good call, I updated the ticket.

**Pull request merged**